src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import { TodoContext } from '../../context/TodoContext';
+import { deleteTodos } from '../../api/todos';
+import { Error } from '../../types/Error';
+import { TodoFilter } from '../TodoFilter';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.